src/main/java/uk/gov/hmcts/reform/juddata/camel/service/AuditProcessingService.java

```diff
@@ -43,39 +39,23 @@
     @Qualifier("springJdbcTransactionManager")
     PlatformTransactionManager platformTransactionManager;
 
```

Comment:
could you please create  an interface  for AuditProcessingService  and declare methods  and AuditProcessingServiceImpl class do the implementation ?

Reply:
yes correct. but some in-cosistancey may services from start without interface. will add but it will make incositant.

Reply:
i am not understanding about your concern. That is the best practise while doing coding